src/pages/home/HomeBlogPreview.jsx

```diff
@@ -336,16 +334,16 @@ const handleImageLoad = (path) => {
   }
 };
 
-export function FeaturedBlogPreview({ blogs, width, imageHeight }) {
+export function FirstBlogPreview({ blogs, width, imageHeight }) {
```

Comment:
```suggestion
export function FeaturedBlogPreview({ blogs, width, imageHeight }) {
```
I would also return this to its original name, though your name change wasn't a bad idea